src/components/layout/AppBar.vue

```diff
@@ -231,6 +255,21 @@ export default class AppBar extends Mixins(StateMixin, ServicesMixin) {
     return this.topNavPowerDevice?.status === 'on'
   }
 
+  get topNavPinLight (): string {
+    return this.$store.state.config.uiSettings.general.topNavPinLight
+  }
+
+  isPinLightON () {
+    const pins : Array<OutputPin> = this.$store.getters['printer/getPins']
+    const lightPin = pins.find(pin => { return pin.name === this.topNavPinLight })
+    if (lightPin !== undefined) return lightPin.value === 1
+    return false
+  }
```

Comment:
This should probably be a getter, and I think it can all be moved into `topNavPinLightOn` getter.